### PR TITLE
Registrants: sorting PDF exports by last name

### PR DESCRIPTION
**Pull request by pferreir**

### CODE REVIEW

File: indico/modules/events/registration/controllers/management/reglists.py [hunk 1]
```diff
@@ -418,15 +418,14 @@ def _checkParams(self, params):
...
-                              .all())
+                              .with_parent(self.regform))
 
 
 class RHRegistrationEmailRegistrantsPreview(RHRegistrationsActionBase):
     """Previews the email that will be sent to registrants"""
 
     def _process(self):
-        registration = self.registrations[0]
+        registration = self.registrations.find_first()
```
Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/reglists.py]: I don't think that'll work. `find_first` is only defined on the models, not on query objects. `.first()` should work though

**Comment by pferreir** [comment]
Superseded by https://github.com/indico/indico/pull/2171

File: indico/modules/events/registration/controllers/management/reglists.py (hunk 1 above)
Comment (pferreir) [inline indico/modules/events/registration/controllers/management/reglists.py]: ... and you're right, of course. I wonder how I managed to test sending mails, if that should break. Weird.

Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/reglists.py]: It's the email *preview*

**Pull request closed without merging**